submissions/Eugene-CG/memory-pair-game/index.js

```diff
@@ -0,0 +1,71 @@
+const cardsList = document.querySelector(".cards__list");
+let clickRemember = [];
+class Card {
+  constructor(img) {
+    this.img = img;
+  }
+  createCard() {
+    return `
+    <li class="card__container">
+          <div class="card card-back">
+              <img src="./img/${this.img}">
+          </div>
+      </li>`;
+  }
+}
+const startGame = () => {
+  let cards = [
+    "fool.jpg",
+    "lovers.jpg",
+    "magician.jpg",
+    "priestess.jpg",
+    "star.jpg",
+    "sun.jpg",
+    "hanged.jpg",
+    "justice.jpg",
+  ];
+  cards = [...cards, ...cards];
+  shuffleCards(cards);
+  cards = createCardOjects(cards);
+  createAllCards(cards);
+};
+const shuffleCards = (cards) => cards.sort(() => Math.random() - 0.5);
+const createCardOjects = (cards) => cards.map((imgSrc) => new Card(imgSrc));
+const createAllCards = (cards) =>
+  cards.forEach((card) => (cardsList.innerHTML += card.createCard()));
+
+const flipCard = ({ target }) => {
+  if (target.closest(".card") || target.closest("img")) {
+    clickRemember = [...clickRemember, target.firstElementChild];
+    target.closest(".card").classList.remove("card-back");
```

Comment:
Instead of running search in DOM for every card, you can save it into variable inside listener